src/utils/sorting.ts

```diff
@@ -0,0 +1,23 @@
+import {YoroiWallet} from '../yoroi-wallets'
+import {TokenInfo} from '../yoroi-wallets/types'
+
+export const sortTokenInfos = ({wallet, tokenInfos}: {wallet: YoroiWallet; tokenInfos: Array<TokenInfo>}) =>
```

Comment:
@wolverineks in my head is something that we should export within `yoroi-wallets`, so we sort on all platforms in the same way. Maybe moving the `/utils` to `yoroi-wallets/` ?